### DISCUSSION TITLE
What are the limitations between Home, Pro, and Enterprise in terms of what SophiApp can change?

### DISCUSSION THREAD

**Discussion by tdbe** [Sophia-Community/SophiApp#142] [Q&A] [answered]
### Your Idea

I am coming from the context of  windows 10 and other older tools to decrapify windows. It was historically the case that you could not decrapify much if you were stuck with Windows Home, and the best one was Enterprise because of group policy tweaks which are very powerful & resilient.

Is there a similar pattern with SophiApp? For example is there anything SophiApp can't do in Windows 11 Home, that it can do in Windows 11 Enterprise?

Furthermore (I don't know yet much about how SophiApp works), are there any things that MS might be able to bring back / reset via updates / backdoors etc, and undo any of the things SophiApp did? (do you need to re-run it periodically or does it run itself and keep its changes up?)

Perhaps clarifying some of these questions in the readme will help others.

Thanks!

### Windows Version

Windows 11

**Comment by farag2** [comment, marked as answer]
Hi Thank you for your question. Let's start with the wording `decrapify windows`. I really hate it. It sounds like Windows contains `crap` or something we want to get rid of. This is not true. Windows is an Enterprise OS by a twist of fate became very `home` world-wide OS. So we (and I especially) prefer the word `customization`. It means that we allow users to fine-tune their OS without making any harm to the system.

> Is there a similar pattern with SophiApp? For example is there anything SophiApp can't do in Windows 11 Home, that it can do in Windows 11 Enterprise?

Pretty hard to answer even in Russian for me. :) SophiApp differentiates Windows' versions but every function works and is applicable for any editions, except very few. may be they're related with some telemetry disabling. So you can see every function in the UI anyway. If your Windows doesn't support some, this function will be disabled to apply and greyed out. So really do not care about editions.

> backdoors

Oh, very bad word towards Windows and MS. It sounds like Microsoft spies on you. xD Like many kiddies in the Internet say. :D Of course this is not true.

> are there any things that MS might be able to bring back

Windows really reverts some changes made by the app (I saw it on my Windows 11 after the upgrade up to 22621 build). The first part of them makes changes to the registry by restoring some keys that were removed. This is `normal` keys which can be removed. We do not break Windows via the or the script. But some keys were reverted for too for the unknown reason: the file transfer dialog box was collapsed. That's all. So I mean all reverted changes can be explained. None of policies or things was touched.

> do you need to re-run it periodically

For your peace of mind, I'd advise to apply it again only after major upgrades like it was on September, 20th. No need to apply script after every update. It makes no sense. Regarding SophiApp, it shows the real functions' state in the UI. So you may check, what's changed. But I've never seen if anything was resetted after a minor update.

> Perhaps clarifying some of these questions in the readme will help others.

Do a PR. 🐱 

Feel free to ask anything. Or in our [Telegram group](https://t.me/sophia_chat).

P.S. We start developing from the scratch SophiApp 2.0 with fully new UI/UX. ETA? No ETA when the war's knocking your door. Somewhere in 2023. 😺

**Reply by tdbe** [reply]
Thanks a lot for the detailed reply. Also paints a good picture for the project. Customizations wise it is nice, neat and simple for the user.  👍 Also looks like it's proudly & passionately developed.

Just to keep the public fairly informed: I wouldn't paint MS's windows in a better light than it objectively deserves in terns of crap (e.g. ads, shovelware), backdoor access (connects to internet before login, "restores" user's explicitly refused consent & defaults, mysteriously re-installs and/or reconfigures "crucial components" such as edge & cortana), telemetry (uploads user activity & which apps you have, stores & uploads your clipboard history to the cloud as a "feature"), integrates its own apps nobody wants in ways that gets them repeatedly sued for antitrust etc... 

All this and they still can't fix eg the sound control panel that is unchanged since windows 95.
